src/contracts/security/security.service.spec.ts

```diff
@@ -223,11 +225,11 @@ describe('SecurityService', () => {
       const mockProviderCall = jest
         .spyOn(providerService.provider, 'call')
         .mockImplementation(async () => {
-          const iface = new Interface(SecurityAbi__factory.abi);
-          return iface.encodeFunctionResult('isPaused', [expected]);
+          const iface = new Interface(StakingRouterAbi__factory.abi);
+          return iface.encodeFunctionResult('isActive', [expected]);
```

Comment:
Does this test pass? the request calls `isActive` method instead of `isPaused`, there should be reverse logic here: `iface.encodeFunctionResult('isActive', [!expected]);`